models/Event.js

```diff
@@ -46,13 +46,13 @@ const EventSchema = new Schema({
 		type: Number,
 		required: true
 	},
-	type_id: {
-		type: Number,
-		required: true
+	type: {
```

Comment:
now that im thinking about it, im wondering if we need to be careful about using the word "type". maybe we should call it "category" instead, and also, it might fit better into the definition for what this field is representing

Reply:
`type` changed to `category`